### ISSUE TITLE
colons "always" check fails

### ISSUE THREAD

**Issue by olegskl** [rossPatton/stylint#123]
In Stylint version: 1.0.4 on OSX, Windows, ...
Given: `"colons": {"expect": "always", "error": true},`

We get:

> Error: unecessary colon found
File: app/bundles/logged-site/logged-site.styl
Line: 2: background: yellow;

Most probably because starting from [line 19 in checks/colons.js](https://github.com/rossPatton/stylint/blob/b9c3219f0c12813a8573358acab0887fe7becb18/src/checks/colons.js#L19) we have:

```js
var arr = line.split( ' ' )
if (... && arr[0].indexOf( ':' ) === -1) { colon = false }
```

So:

```js
var arr = '  background: yellow'.split( ' ' ) // ["", "", "background:", "red"]
// and colon becomes false
```

I think that the regression has appeared in 1bb2eee1ebfd4625916c14e446501ddf7e85efec and there are no tests for untrimmed lines.

**Comment by rossPatton** [comment]
i had a utility function that trims the lines that i wasn't using. 1.0.6 should resolve this

**Comment by olegskl** [comment]
Seems to be working in 1.0.6.

**Comment by rossPatton** [comment]
awesome, closing out...

**Comment by Alxblsk** [comment]
Hi

I found a weird issue with colon detection:

Code:
.smart-image-gallery {
    .test2 {
        display: flex;
        flex-flow: column nowrap;
        justify-content: space-between;
        align-items: center;
    }
}

Error:
>Error: missing colon between property and value
>File: src/gallery.styl
>Line: 2: .test2 {

I found that if I rewrite '.smart-image-gallery' with just '.smart', I won't receive this error.
Could you take a look?
